### PR TITLE
fix up travis deploy config

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
move distributions key to the right place
install setuptools_scm for deploy

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,15 +37,15 @@ jobs:
     - stage: deploy
       python: '3.6'
       env:
-      install: skip
+      install: pip install -U setuptools setuptools_scm
```
Comment (nicoddemus) [inline .travis.yml]: Why this change? I think we don't need this because this dependencies are already configured in `setup.py`.

Comment (RonnyPfannschmidt) [inline .travis.yml]: well - without this, we trigger an easy_install, pep-518 isnt live yet ^^

Comment (nicoddemus) [inline .travis.yml]: I see, thanks.
